### ISSUE TITLE
php7.2 support

### ISSUE THREAD

**Issue by nyetwurk** [kawf/kawf#46]
the scripts seem to work with php7 but I have not tested apache-php7 mod yet.

**Comment by nyetwurk** [comment]
each() is deprecated.
https://wiki.php.net/rfc/deprecations_php_7_2#each
```
PHP Deprecated:  The each() function is deprecated. This message will be suppressed on further calls in /var/www/kawf/include/template.inc on line 145
```

**Comment by nyetwurk** [comment]
fixed between 6fc5b9f86be0b34956ee600148772850e3b51593 and f18680e649412778fe3f0d7c7397e53c2fc5cce5
